pillar.example

```diff
@@ -34,6 +34,12 @@ postgresql:
     wal_keep_segments: (unset)
     wal_buffers: (unset)
 
+    log_statement: 'none'
```

Comment:
I think the `''` should be removed here. It's a double escape and there's no value for `log_statement` which needs escaping.